NPCs.bb

```diff
@@ -3037,7 +3040,10 @@ Function UpdateNPCs()
 							n\State = n\State-FPSfactor
 						EndIf
 						n\State=n\State-(FPSfactor/80.0)
-						If n\State <= 0 Then n\Idle = True	
+						If n\State <= 0 Then
+							n\Idle = True
+							PositionEntity(n\Collider, 0, 500, 0)
+						EndIf	
```

Comment:
We teleport 372 into the abyss when it's idle so its hidden collider doesn't show up on the NVG interface. These coords are the same ones 106 gets teleported to when idle.

I had wanted to use the `HideFromNVG` property of NPCs for this purpose but that causes the NPC to immediately disappear from the NVG interface rather than at the next refresh.